product_docs/docs/biganimal/release/overview/03_security/01_securing_your_cluster.mdx

```diff
@@ -0,0 +1,175 @@
+
+Before deploying your clusters, EDB recommends that you perform the following steps in each of your Azure subscriptions:
+
+## Configure the default Azure Security Center (ASC) policy 
+
+Define the parameters in your default ASC policy to ensure that your resources are compliant with BigAnimal's security requirements.  
+
+#### Allowed container ports list 
+BigAnimal runs containers using these ports: 
+ * 5432
+ * 9187
+ * 8000
+ * 9000
+ * 9443
+ * 9402
+ * 10250
+ * 9090
+ * 8080
+ * 8443
+ * 2020
+ * 3000
+ * 6443
+ * 9100
+ * 9102
+ * 9201
+ * 8081
+
+#### Recommended Allowed service ports list 
+
+BigAnimal runs containers using these services ports:
+ * 5432
+ * 9402
+ * 443
+ * 8080
+ * 9090
+ * 3000
+ * 8443
+ * 9443
+ * 9100
+ * 9201
+
+
+#### Recommended Allowed AppArmor profiles
+
+BigAnimal uses AppArmor's default profile: 
+ * runtime/default
+
+#### Recommended Allowed capabilities 
+
+BigAnimal runs containers with minimum security capabilities only required to implement data plane functionality such as logging functionality.
+
+ * FOWNER 
+
+#### Recommended Max allowed memory bytes in Kubernetes cluster
+
+BigAnimal runs containers with a maximum allowed memory bytes of 6Gi.
+
+#### Recommended Max allowed CPU units in Kubernetes cluster
+
+BigAnimal runs containers with a maximum of two allowed CPU units 2. 
+
+#### Recommended Allowed host paths 
+
+BigAnimal uses the following hotpaths in its containers:
+
+```
+{
+ "paths": [
+   {
+     "pathPrefix": "/var/log",
+     "readOnly": false
+   },
+   {
+     "pathPrefix": "/var/lib/docker/containers",
+     "readOnly": true
+   },
+   {
+     "pathPrefix": "/",
+     "readOnly": true
+   },
+   {
+     "pathPrefix": "/sys",
+     "readOnly": true
+   },
+   {
+     "pathPrefix": "/proc",
+     "readOnly": true
+   },
+   {
+     "pathPrefix": "/var/run/docker.sock",
+     "readOnly": false
+   },
+   {
+     "pathPrefix": "/run/containerd/containerd.sock",
+     "readOnly": false
+   },
+   {
+     "pathPrefix": "/dev",
+     "readOnly": false
+   },
+   {
+     "pathPrefix": "/boot",
+     "readOnly": true
+   },
+   {
+     "pathPrefix": "/lib/modules",
+     "readOnly": false
+   },
+   {
+     "pathPrefix": "/usr",
+     "readOnly": true
+   },
+   {
+     "pathPrefix": "/etc",
+     "readOnly": true
+   }
+ ]
+}
+```
+#### To configure the ASC policy parameters
+
+1. Launch the Azure Policy service in the Azure portal by searching for and selecting the *Policy* service in the search box at the top.
+
+1. Select **Assignments** on the left side of the Azure Policy page. 
+
+1. Select **Assign Policy** from the top of the Policy - Assignments page.
+
+1. On the Assign Policy page, set the Scope by selecting the ellipsis and then selecting the applicable subscription. A scope determines what resources or grouping of resources the policy assignment gets enforced on. Then use the **Select** button at the bottom of the Scope page.
```

Comment:
Ellen Liu can you provide information on whether these steps are required for a user performing the security centre tweaks for the first time?

Reply:
After having access to the Dev build i am able to work this out. Marking this as resolved.